tests/utilities/test_auto_restart.py

```diff
@@ -205,12 +214,21 @@ def test_fast_forward_on_random_sampler():
 
 class RangeIterableDataset(IterableDataset):
 
-    def __init__(self, data, num_workers: int, batch_size: int, is_in_workers: bool, state_dict=None):
+    def __init__(
+        self,
+        data,
+        num_workers: int,
+        batch_size: int,
+        is_in_workers: bool,
```

Comment:
What is `is_in_workers` for?

Reply:
This was used with previous version of the PoC. Can be removed, we managed to remove the need for this variable.